docs/operating-modes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Operating modes
+sidebar_position: 25
+custom_edit_url: https://github.com/stryker-mutator/stryker-net/edit/master/docs/operating-modes.md
+---
+# Operating modes
+
+There are currently two different ways to run Stryker:
+
+- From the test project context
+- From the target project context
+
+You will also be able to run Stryker from your solution context in the future.
+
+## Test project context
+
+Just run
+
+```bash
+dotnet stryker
+```
+
+in your test project folder. Stryker will automatically detect the target project based on the project reference. If your test project has multiple project references, you need to specify the target project with `--project <filename>` ([docs here](https://stryker-mutator.io/docs/stryker-net/configuration/#project-file-name)). You can only specify one target project at the time.
```

Comment:
```suggestion
Just run `dotnet stryker` in your test project folder.  
Stryker will automatically detect the target project based on the project reference. If your test project has multiple project references, you need to specify the target project with `--project <filename>` ([docs here](https://stryker-mutator.io/docs/stryker-net/configuration/#project-file-name)). You can only specify one target project at the time.
```

Reply:
Hier ook source project noemen dan?

```suggestion
Just run `dotnet stryker` in your test project folder.  
Stryker will automatically detect the source project based on the project reference. If your test project has multiple project references, you need to specify the source project with `--project <filename>` ([docs here](https://stryker-mutator.io/docs/stryker-net/configuration/#project-file-name)). You can only specify one source project at the time.
```

Reply:
If @richardwerkman agrees with that naming